docs/extensions/canonical_tutorial.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from sphinx.util.docutils import SphinxDirective
+
+if TYPE_CHECKING:
+    from typing import ClassVar
+
+    from docutils import nodes
+    from sphinx.application import Sphinx
+
+
+class CanonicalTutorial(SphinxDirective):
```

Comment:
Why does this need to be duplicated

Reply:
it’s not. this is the version that does nothing (because this *is* the canonical location).

in the scanpy-tutorials repo, it displays a box linking to the version here.